spirv_msl.cpp

```diff
@@ -1512,6 +1512,11 @@ void CompilerMSL::preprocess_op_codes()
 	    (is_sample_rate() && (active_input_builtins.get(BuiltInFragCoord) ||
 	                          (need_subpass_input && !msl_options.use_framebuffer_fetch_subpasses))))
 		needs_sample_id = true;
+
+	if (preproc.ray_tracing)
```

Comment:
Should validate MSL version somewhere. This seems like a natural place to do so.